package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "photon-extension-kit",
+  "version": "0.0.1",
+  "description": "Photon styles for your brand new Firefox extension.",
+  "main": "script.js",
+  "dependencies": {
+    "photon-colors": "^2.0.1"
```

Comment:
Just `"2.0.1"` please. I want to make sure we use the same version everywhere. :smile: